### DISCUSSION TITLE
Access denied

### DISCUSSION THREAD

**Discussion by yanboit** [qkqpttgf/OneManager-php#698] [Q&A]
这是什么情况，刚开始有用，后面就这样了，感觉新版才有这样的问题
![Uploading Screenshot_2023-08-08-09-27-12-652_com.android.chrome.jpg…]()

**Comment by qkqpttgf** [comment]
呃，你这图片，我各种姿势都没能打开……

**Reply by yanboit** [reply]
github的问题吧重新上传了，我重装后好了，之前换服务器直接打包解压的，给目录权限了，可以打开过会儿就这个样子了，后面重装就好了
![Screenshot_2023-08-08-09-27-12-652_com android chrome](https://github.com/qkqpttgf/OneManager-php/assets/72846735/17f74295-71a0-4ee9-92f8-2910da6f508f)

**Comment by qkqpttgf** [comment]
那肯定是你家BT干的

**Reply by yanboit** [reply]
有可能吧，我换了个面板，可能就出现这样问题了，重装没问题
